server/server.js

```diff
@@ -222,34 +222,44 @@ searchFilter: function (option) {
 
     console.log(selectorCategory);
 
-      var query = Listing.find({ $and:[ selectorCategory, selectorCity ] }).fetch();
-      console.log(query);
-      return query;
-},
-addOffer: function (options) {
-  Offer.insert({
-    listing_title: options.listing_title,
-    img: options.img,
-    offerprice: options.offerprice,
-    date: options.date,
-    location: options.location,
-    listingId: options.listingId,
-    lat: options.lat,
-    lng: options.lng,
-    creator_id: options.creator_id,
-    status: options.status
+    var query = Listing.find({ $and:[ selectorCategory, selectorCity ] }).fetch();
+    console.log(query);
+    return query;
+  },
+  addOffer: function (options) {
+    Offer.insert({
+      listing_title: options.listing_title,
+      img: options.img,
+      offerprice: options.offerprice,
+      date: options.date,
+      location: options.location,
+      listingId: options.listingId,
+      lat: options.lat,
+      lng: options.lng,
+      creator_id: options.creator_id,
+      status: options.status
+    });
+  },
+  addProfileInfo: function (options) {
+    Meteor.users.update( this.userId, { $set: {
+      'profile.photo': options.photo,
+      'profile.about': options.about,
+      'profile.email': options.email,
+      'profile.location': options.location,
+      'profile.link': options.link 
+    }});
+  },
+// IDEAL: instead of manually typing in keys, use iteration
+addReport: function (options){
+  Report.insert({
+    prohibited_box: options.prohibited_box,
+    offensive_box: options.offensive_box,
+    irrelevant_box: options.irrelevant_box,
+    false_box: options.false_box,
+    compliance_box: options.compliance_box,
```

Comment:
Identifier 'compliance_box' is not in camel case.<br>Too many errors. (83% scanned).